src/main/java/com/bynder/sdk/query/MediaModifyQuery.java

```diff
@@ -112,6 +123,7 @@ public MediaModifyQuery setMetaproperties(List<MetapropertyAttribute> metaproper
         return this;
     }
 
+    public String getArchiveDate() { return archiveDate; }
     public Boolean getLimited() {
```

Comment:
Can you please add a `\n` for formatting
```suggestion
    public String getArchiveDate() { return archiveDate; }
    
    public Boolean getLimited() {
```

Reply:
@Arpit-Sharma-USC  - added a line break.